src/app/components/team/TiplineInbox.js

```diff
@@ -30,32 +28,28 @@ const TiplineInbox = ({ routeParams }) => (
         if (!error && props) {
           const { team } = props;
           const defaultStatusId = team.verification_statuses.default;
+          // Should we discard savedQuery already?
```

Comment:
Yes, we can get rid of `savedQuery` already.

Reply:
Done